src/routes/become-a-provider/[[network=networks]]/+layout.svelte

```diff
@@ -0,0 +1,7 @@
+<script>
```

Comment:
note the path to this file, it uses the `match` function above to figure out if the route is valid, and then populates $page.params accordingly with a value for `network`, or not.